salt/modules/boto_elbv2.py

```diff
@@ -72,6 +72,70 @@ def __virtual__():
     return True
 
 
+def create_target_group(name, protocol, port, vpc_id,
+                        region=None, key=None, keyid=None, profile=None,
+                        health_check_protocol='HTTP', health_check_port='traffic-port',
+                        health_check_path='/', health_check_interval_seconds=30,
+                        health_check_timeout_seconds=5, healthy_threshold_count=5,
+                        unhealthy_threshold_count=2):
```

Comment:
We need some documentation added below which explains what each of these arguments do, please.